lib/user_client.dart

```diff
@@ -1,41 +1,35 @@
-import 'package:flutter/material.dart';
-import 'package:flutter/services.dart';
-import 'package:onegini/onegini_event_listener.dart';
-import 'package:onegini/pigeon.dart';
+import 'dart:async';
 
-import 'constants/constants.dart';
-import 'onegini.dart';
+import 'package:onegini/events/onewelcome_events.dart';
+import 'package:onegini/pigeon.dart';
 
 ///Сlass with basic methods available to the developer.
 class UserClient {
   final UserClientApi api;
   UserClient(this.api);
 
+  // ignore: close_sinks
+  final owEventStreamController = StreamController<OWEvent>.broadcast();
```

Comment:
This comment was added by me for convenience, can you investigate if we should actually close it?

Reply:
added finalizer to close the broadcaster before the onegini instance dies